shared/validation/languages/ne.js

```diff
@@ -0,0 +1,53 @@
+import tokenizeWords from 'talisman/tokenizers/words';
+
+// Minimum of words that qualify as a sentence.
+const MIN_WORDS = 1;
+
+// Maximum of words allowed per sentence to keep recordings in a manageable duration.
+const MAX_WORDS = 10;
+
+// Numbers that are not allowed in a sentence depending on the language. For
+// English this is 0-9 once or multiple times after each other.
+const NUMBERS_REGEX = /[०-९0-9]+/;
+
+// Checks whether the sentence has a ? or । character in the middle, as it could mean
+/* eslint-disable-next-line no-unused-vars */
+const STRUCTURE_REGEX = /[?।!].+/;
+
+// The following symbols are disallowed, please update here as well and not just the regex
+// to make it easier to read:
+// < > + * \ # @ ^ [ ] ( ) /
+/* eslint-disable-next-line no-useless-escape */
+const SYMBOL_REGEX = /[<>\+\*\\#@\^\[\]\(\)\/]/;
+// Any words consisting of uppercase letters or uppercase letters with a period
+// inbetween are considered abbreviations or acronyms.
+// This currently also matches fooBAR but we most probably don't want that either
+// as users wouldn't know how to pronounce the uppercase letters.
+const ABBREVIATION_REGEX = /[A-Z]{2,}|[A-Z]+\.*[A-Z]+/;
+
+const ENGLISH_ALPHABETS_REGEX = /^[a-zA-Z]+$/;
```

Comment:
this won't match anything with a space in it, or any punctuation at the end of the sentence. Here are some examples:

```
This is a sentence
This is a sentence.
This is a sentence?
ThisIsASentence.
```

Reply:
Okay. Let me check these things.

Reply:
I am actually not allowing english alphabets at all when nepali language is selected. Not only the sentences with space or punctuation, but sentence with english alphabets are not allowed with the current implementation. 
  I tested with other language. It's working fine. 
@MichaelKohler

Reply:
Have you also tested with my example sentences and selecting Nepali? None of these match and therefore are valid. If you don't want to allow any English characters, you might use `/[a-zA-Z]/` without the `^` and `$`. This would then match always when there is an English alphabet character. Otherwise it will only match when there are no spaces or anything else, basically only if there are a-z/A-Z multiple times.

Reply:
I got it now. Thanks for the explanation. I have made the changes accordingly. :) Tested for the above examples and few others.